.github/workflows/ci.yml

```diff
@@ -11,7 +11,7 @@ jobs:
         with:
           distribution: 'temurin'
           java-version: 21
-      - uses: gradle/actions/setup-gradle@d9c87d481d55275bb5441eef3fe0e46805f9ef70 # v3.5.0
+      - uses: gradle/actions/setup-gradle@af1da67850ed9a4cedd57bfd976089dd991e2582 # v4
```

Comment:
Converted to draft, because this `v4` currently points to pre-release https://github.com/gradle/actions/releases/tag/v4.0.0-rc.1